operators/pkg/tenant-controller/tenant_controller.go

```diff
@@ -296,7 +298,10 @@ func (r *TenantReconciler) createOrUpdateClusterResources(ctx context.Context, t
 		ObjectMeta: metav1.ObjectMeta{Name: "crownlabs-resource-quota", Namespace: nsName},
 	}
 	rqOpRes, err := ctrl.CreateOrUpdate(ctx, r.Client, &rq, func() error {
-		r.updateTnResQuota(&rq)
+		rq.Labels = r.updateTnResourceCommonLabels(rq.Labels)
+		tn.Status.Quotas = forge.TenantResourceList(workspaces)
```

Comment:
This function should mutate the resource quota only. Tenant modifications should be performed in advance.